docs/admin/Nuxwdog.md

```diff
@@ -0,0 +1,104 @@
+Nuxwdog
+=======
+
+## Overview
+
+Nuxwdog is a script that can be used to collect relevant passwords before a server starts.
+It utilizes the `keyutils` package and caches the password on Kernel Keyring.
+
+Nuxwdog is used in Red Hat Certificate System to collect passwords before the PKI Server starts.
```

Comment:
Do not mention downstream products in upstream docs.

Reply:
I agree with edewata here.